inc/class-wpseo-replace-vars.php

```diff
@@ -413,7 +413,7 @@ private function retrieve_sitedesc() {
 		static $replacement;
 
 		if ( ! isset( $replacement ) ) {
-			$description = trim( strip_tags( get_bloginfo( 'description' ) ) );
+			$description = trim( wp_strip_all_tags( get_bloginfo( 'description' ) ) );
```

Comment:
As `wp_strip_all_tags` already trims the content, this does not have to be wrapped again.